integration/test/Test/Conversation.hs

```diff
@@ -1112,3 +1112,17 @@ testReplaceMembersConvNotFoundOtherDomain = do
   bindResponse (replaceMembers alice fakeConv def {users = [bobId]}) $ \resp -> do
     resp.status `shouldMatchInt` 422
     resp.json %. "label" `shouldMatch` "federation-not-implemented"
+
+testReplaceMembersEmpty :: (HasCallStack) => App ()
+testReplaceMembersEmpty = do
+  [alice, bob, charlie, dylan] <- createAndConnectUsers [OwnDomain, OwnDomain, OwnDomain, OwnDomain]
+  conv <- postConversation alice (defProteus {qualifiedUsers = [bob, charlie]}) >>= getJSON 201
+
+  -- Replace members: remove everyone
+  bindResponse (replaceMembers alice conv def {users = []}) $ \resp -> do
+    resp.status `shouldMatchInt` 200
+
+  -- Verify conversation members
+  bindResponse (getConversation dylan conv) $ \resp -> do
+    resp.status `shouldMatchInt` 403
+    resp.json %. "label" `shouldMatch` "access-denied"
```

Comment:
This test might have some value, however, it is not the correct use case. We need to create a team channel where the admin is not a member of instead of a non-team proteus conversation. See `testTeamAdminCanReplaceMembers` in integration/test/Test/Channels.hs`. Then check the actual member list of the channel.